pkg/maps/lbmap/lbmap.go

```diff
@@ -610,10 +610,20 @@ func (svcs svcMap) addFEnBE(fe *loadbalancer.L3n4AddrID, be *loadbalancer.Backen
 	return &lbsvc
 }
 
-// InitMapInfo updates the map info defaults for sock rev nat {4,6} and lb maps.
-func InitMapInfo(maxSockRevNatEntries, lbMapMaxEntries int) {
+// Init updates the map info defaults for sock rev nat {4,6} and LB maps and
+// then initializes all LB-related maps.
+func Init(params InitParams, maxSockRevNatEntries, lbMapMaxEntries int) {
```

Comment:
One more nit: could we move `maxSockRevNatEntries` and `lbMapEntries` to `InitParams`? Having them in the struct will give us guarantee that in the future we won't mix values. It could be done as a follow-up.

Reply:
Done